pkg/flow/ent/schema/instance.go

```diff
@@ -27,6 +27,7 @@ func (Instance) Fields() []ent.Field {
 		field.String("errorCode").Optional(),
 		field.String("errorMessage").Optional(),
 		field.String("invoker").Optional(),
+		field.Strings("parents").Optional(),
```

Comment:
If we're going to add a field here purely for the purpose of adding tags to logs, I think we should make it more broadly suited for that purpose. So that it can contain all the extra tags, even ones we haven't thought of yet, without needing to add more fields to the table. Maybe rename this column `tags` and make it a big json struct?